server/src/controllers/tasks/editTaskController.ts

```diff
@@ -0,0 +1,17 @@
+import { Request, Response, NextFunction } from 'express';
+import { editTaskQuery } from '../../database';
+
+const editTask = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const { id } = req.params;
+    const {
+      name, description, priority, endDate, status,
+    } = req.body;
+    const { rows } = await editTaskQuery(+id, name, description, priority, endDate, status);
```

Comment:
when we have one or two arguments we pass it like this, but when we have more than two arguments, we should pass them as object.